test/format/failure_configs/CONFIG.WT-15826

```diff
@@ -0,0 +1,212 @@
+############################################
```

Comment:
I would create a seperate ticket to add this, once disagg leader mode tests are stable, otherwise this could cause noise on mainline.